sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -220,4 +220,12 @@ class SparkSqlParserSuite extends PlanTest {
 
     intercept("explain describe tables x", "Unsupported SQL statement")
   }
+
+  test("SPARK-18106 analyze table") {
```

Comment:
Then maybe you can move those parse tests here ? All I'm suggesting is that the parse tests all be together.